web/packages/build/vite/config.ts

```diff
@@ -69,6 +69,18 @@ export function createViteConfig(
         outDir: outputDirectory,
         assetsDir: 'app',
         emptyOutDir: true,
+        rollupOptions: {
+          output: {
+            // This removes code splitting.
+            inlineDynamicImports: true,
```

Comment:
Can we remove `React.lazy` and all the default exports instead? Otherwise we're wrapping the loading of all route components in a promise for no reason

Reply:
Removed react lazy on everything except access graph/assist. Was getting really weird "React Markdown" errors unless it was lazy loaded. like 40+ of them during testing. I'll keep inspecting but for now, every other lazy is removed

Reply:
We could something like with `manualChunks` to programmatically choose which chunks are split and remove hashing that way for those specifically, or we can keep this line to handle the assist/access graph and call it a day. I dont know much about assist's internals I think to know what to chunk out

Reply:
e update if we stick with these changes https://github.com/gravitational/teleport.e/pull/3566